packages/aws-cdk-lib/aws-rds/lib/instance-engine.ts

```diff
@@ -1841,67 +2249,159 @@ class OracleEeCdbInstanceEngine extends OracleInstanceEngineBase {
  * and `DatabaseInstanceEngine.sqlServerEe`).
```

Comment:
```suggestion
 * and `DatabaseInstanceEngine.sqlServerEe`).
 *
 * @see https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_SQLServer.html#SQLServer.Concepts.General.VersionSupport
```
I think it would be useful to add links in the documentation for each class showing the version notes for each instance and cluster engine.

Reply:
Seen as the documentation doesn't seem to match the live data, it might be confusing, not sure about this

Reply:
Yup, that's confusing indeed. I guess we should trust the API.
Thanks for the detailed clarification 👍